skbio/sequence/_nucleotide_mixin.py

```diff
@@ -230,6 +230,87 @@ def is_reverse_complement(self, other):
             # underlying sequence data
             return self.reverse_complement()._string == other._string
 
+    def gc_content(self):
+        """Calculate the relative frequency of G's and C's in the sequence.
+
+        This includes G, C, and S characters. This is equivalent to calling
+        ``gc_frequency(relative=True)``.
```

Comment:
Can you add a note here about the gap character behavior? In case the user doesn't read `gc_frequency` docs.